nextcord/raw_models.py

```diff
@@ -168,7 +169,17 @@ class RawReactionActionEvent(_RawReprMixin):
         .. versionadded:: 1.3
     """
 
-    __slots__ = ("message_id", "user_id", "channel_id", "guild_id", "emoji", "event_type", "member")
+    __slots__ = (
+        "message_id",
+        "user_id",
+        "channel_id",
+        "guild_id",
+        "emoji",
+        "event_type",
+        "member",
+        "burst",
+        "burst_colors",
```

Comment:
There seems to be a missing [type attribute](https://discord.com/developers/docs/resources/message#get-reactions-reaction-types) here, although I'm not sure why it exists since the `burst` boolean is also there...